playwright/src/main/java/com/microsoft/playwright/junit/impl/ExtensionUtils.java

```diff
@@ -49,6 +51,13 @@ static boolean isParameterSupported(ParameterContext parameterContext, Extension
   static void setTestIdAttribute(Playwright playwright, Options options) {
     String testIdAttribute = options.testIdAttribute == null ? "data-testid" : options.testIdAttribute;
     playwright.selectors().setTestIdAttribute(testIdAttribute);
+  }
 
+  private static Class<?> getOuterClass(Class<?> clazz) {
+    if (clazz.getEnclosingClass() == null) {
+      return clazz;
+    } else {
+      return getOuterClass(clazz.getDeclaringClass());
```

Comment:
Let's use either getDeclaringClass() or getEnclosingClass() in both branches, otherwise it might throw NPE.

Reply:
ah i missed that, thanks for catching/fixing